SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -972,7 +972,12 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment(FString ForceSnaps
 		}
 
 		FLocalDeploymentManager::LocalDeploymentCallback CallBack = [this](bool bSuccess) {
-			if (!bSuccess)
+			if (bSuccess)
+			{
+				// Once we've launched the local deployment, attempt to start the inspector process
```

Comment:
Nit: I would suggest this comment does not add meaningful context for the reader. The call to StartInspectorProcess seems informative enough to me.

Reply:
good point, removed